composer.json

```diff
@@ -51,5 +51,10 @@
     },
     "bin": [
         "bin/doctrine-migrations"
-    ]
+    ],
+    "support": {
+        "issues": "https://github.com/doctrine/migrations/issues",
+        "source": "https://github.com/doctrine/migrations",
```

Comment:
these 2 are not necessary, as they are already auto-detected by Composer AFAIK.

Reply:
If what @stof says above is true, then :+1: